cmd/tink-server/main.go

```diff
@@ -93,6 +95,11 @@ func (c *DaemonConfig) PopulateFromLegacyEnvVar() {
 	if basicAuthPass := os.Getenv("TINK_AUTH_PASSWORD"); basicAuthPass != "" {
 		c.HTTPBasicAuthPassword = basicAuthPass
 	}
+	if insecure, isSet := os.LookupEnv("INSECURE"); isSet {
+		if b, err := strconv.ParseBool(insecure); err != nil {
```

Comment:
Should this be `err == nil`?

Or, should this result in a panic?

You could also ignore the error I suppose, as that will result in what I consider to be a decent default: that a TLS listener gets created.

Reply:
I think I cribbed this from ONLY_MIGRATION above. a bug in both locations? Did any of this ever work??

Reply:
It's probably time to collapse the two different codepaths for env vars and command line flags into one piece of correct code... :sigh: